onto_tool/onto_tool.py

```diff
@@ -169,6 +169,8 @@ def generate_graphic(action, onto_files, endpoint, **kwargs):
         Read cached query results
     save_cache: TextIOWrapper
         Save query results as JSON to use with --cache
+    show_bnode_subjects: boolean
+        If true, triples with blank nodes in the subject will be used to generate the graphic
```

Comment:
```suggestion
        If true, triples with blank nodes in the subject will not be filtered out.
```
Otherwise it seems like _only_ blank node subjects will be considered.